components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,40 @@
+import { TooltipPopup, useTooltip } from "@reach/tooltip";
+import "@reach/tooltip/styles.css";
+import { animated, useTransition } from "@react-spring/web";
+import { cloneElement } from "react";
+import styled from "styled-components";
+
+interface Props {
+  "aria-label"?: string;
+  children: React.ReactElement;
+  label: string;
+}
```

Comment:
A simple tooltip component. The child element passed to it will trigger it to show when hovered.